core/engine/src/builtins/function/mod.rs

```diff
@@ -976,6 +976,11 @@ pub(crate) fn set_function_name(
 pub(crate) fn function_call(
     function_object: &JsObject,
     argument_count: usize,
+    #[allow(
+        unused_variables,
+        reason = "Only used if native-backtrace feature is enabled"
+    )]
+    native_source_info: NativeSourceInfo,
```

Comment:
Why not use `InternalMethodContext` instead? Seems better to futureproof any other data added to the internal methods.